fotoapparat/src/main/java/io/fotoapparat/view/CameraView.java

```diff
@@ -52,8 +57,21 @@ public CameraView(@NonNull Context context,
 
     private void init() {
         rendererView = new TextureRendererView(getContext());
-
         addView(rendererView);
+
+        focusMarkerLayout = new FocusMarkerLayout(getContext());
```

Comment:
That's a correct approach, but we can make it even cooler. Instead of adding view here, let's allow user to decide whether he wants to use this functionality or not by adding it to XML layout. So it would look something like that:

```xml
<CameraView>
    <FocusMarkerLayout />
</CameraView>
```